package.json

```diff
@@ -15,8 +15,9 @@
     "watch": "tsc --watch"
   },
   "devDependencies": {
-    "@podman-desktop/api": "1.12.0",
+    "@podman-desktop/api": "^1.16.1",
     "@types/node": "^20",
-    "typescript": "5.5.4"
-  }
+    "typescript": "5.7.3"
+  },
+  "packageManager": "npm@11.0.0+sha512.11dff29565d2297c74e7c594a9762581bde969f0aa5cbe6f5b3644bf008a16c065ece61094d9ffbb81125be38df8e1ba43eb8244b3d30c61eb797e9a2440e3ec"
```

Comment:
Is it really related to the dependencies change or another concern ?

Reply:
to enforce the usage of npm (as there is a package-lock file)

also I read that dependabot should know which package manager to use with this field

Reply:
so either we keep package-lock.json file with packageManager field

or we remove both of them, but we should not have either package-lock without the packageManager field (or the opposite)